Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
```

Comment:
@mreiferson I'm not sure what your stance is on having a Gemfile. And maybe it warrants a PR outside of this one. 
https://help.github.com/articles/using-jekyll-with-pages/#installing-jekyll